config/owasp/suppressions.xml

```diff
@@ -18,4 +18,22 @@
     <cve>CVE-2019-12418</cve>
   </suppress>
 
+  <suppress until="2020-12-31">
+    <notes><![CDATA[Only used in smoke tests]]></notes>
+    <packageUrl regex="true">^pkg:maven/org\.codehaus\.groovy/groovy\-xml@.*$</packageUrl>
```

Comment:
if we `*` every version, shall we reduce the suppression time of `until` then? it's too loose: suppress everything for a year. goes to all instances here. btw, can use regex for groovy ones :wink: